workspace/utilities/lib/util-nav-link-attr.xsl

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+<!-- COMPONENT: util-nav-link-attr -->
+	<xsl:template name="util-nav-link-attr">
+		<xsl:param name="label" select="label" />
+		<xsl:param name="ga-cat" select="'menu-nav'" />
+		<xsl:param name="ga-label" select="concat($url-language, '/', $label)" />
+		<xsl:param name="role" select="'menuitem'" />
+		<xsl:param name="target" select="'_self'" />
+		<xsl:param name="ext-attr" />
+
+		<xsl:variable name="has-ga-cat" select="string-length($ga-cat) != 0" />
+		<xsl:variable name="has-ga-label" select="string-length($ga-label) != 0" />
+		<xsl:variable name="has-role" select="string-length($role) != 0" />
+		<xsl:variable name="has-target" select="string-length($target) != 0" />
+
+		<xsl:if test="$has-ga-cat">
+			<add data-ga-cat="{$ga-cat}" />
```

Comment:
use set instead of add

Reply:
@dominic-blain ^^

Reply:
@nitriques Outdated, was fixed 8 days ago!